server.js

```diff
@@ -28,6 +30,7 @@ module.exports = class Server {
     this.initServer();
     this.initMiddlewares();
     this.initRoutes();
+    getIncrementBalance();
```

Comment:
It's better to create a separate function in server.js called `initCrons`. If you following that `init...` functions pattern that follow it completely